static/app/views/issueList/actions/index.tsx

```diff
@@ -135,14 +139,15 @@ function ActionsBarPriority({
                 onUpdate={handleUpdate}
               />
             </HeaderButtonsWrapper>
-          )}
-          {!anySelected && (
+          ) : organization.features.includes('issue-stream-table-layout') ? (
+            <NarrowHeaderButtonsWrapper>
+              <IssueStreamHeaderLabel>{t('Issue')}</IssueStreamHeaderLabel>
```

Comment:
These headers look a little big - the designs have them at font-size 13px, but fontSizeSmall is probably also fine